src/main/java/com/feedzai/commons/sql/abstraction/dml/dialect/SqlBuilder.java

```diff
@@ -15,14 +15,60 @@
  */
 package com.feedzai.commons.sql.abstraction.dml.dialect;
 
-import com.feedzai.commons.sql.abstraction.ddl.*;
-import com.feedzai.commons.sql.abstraction.dml.*;
+import com.feedzai.commons.sql.abstraction.ddl.AlterColumn;
```

Comment:
Please check before/after.  
Not sure if PDB code style uses wildcard imports or not.